common/settings/settings.ts

```diff
@@ -97,12 +104,15 @@ export enum TokenReadingAnnotation {
 }
 
 export function dictionaryTrackEnabled(dt: DictionaryTrack): boolean {
-    return dt.dictionaryColorizeSubtitles;
+    return dt.dictionaryColorizeSubtitles || dt.dictionaryTokenReadingAnnotation !== TokenReadingAnnotation.NEVER;
 }
 
-export function dictionaryTrackHoverOnly(dt: DictionaryTrack | undefined): boolean {
-    if (!dt) return false;
-    return dictionaryTrackEnabled(dt) && dt.dictionaryColorizeOnHoverOnly;
+export function dictionaryStatusEnabled(dt: DictionaryTrack): boolean {
```

Comment:
Consider renaming to `dictionaryStatusCollectionEnabled`? It took me a little bit to understand the difference between this and `dictionaryTrackEnabled`.